diagnostic.md

```diff
@@ -19,6 +21,8 @@ Record your responses inside the fenced code blocks below each question.
 
     ```md
     <!-- your response here -->
+    component.js - use to define the component, can incorporate classes, tags, actions them.
+    template.hbs - template on how to display data
```

Comment:
Looks good.  The `template.hbs` is also responsible for binding data, and binding and dispatching actions.